tests/test_discoverer.py

```diff
@@ -0,0 +1,21 @@
+import signal
+
+from .setup import discoverer_setup
```

Comment:
If you rename `setup.py` to `conftest.py` you can safely remove this line as the fixtures will be automatically available to you (as far as i remember :))